v2/Makefile

```diff
@@ -259,3 +291,21 @@ staticcheck:
 	if ! which staticcheck >/dev/null; then \
 		env GOFLAGS= go install honnef.co/go/tools/cmd/staticcheck@latest; \
 	fi
+
+define comment_certs
+    $(eval $@_FILE = $(1))
+	@sed -i -E "{s/(^patchesStrategicMerge.*)/# \1/}" ${$@_FILE}
+	@sed -i -E "{s/(^-.*webhook_manifests_patch.*)/# \1/}" ${$@_FILE}
+	@sed -i -E "{s/(^ +files.*)/# \1/g}" ${$@_FILE}
+	@sed -i -E "{s/(^ +.*\.pem.*)/# \1/g}" ${$@_FILE}
+	@sed -i -E "{s/(^ +.*\/tls.*)/# \1/g}" ${$@_FILE}
```

Comment:
I would like to create Secrets with TLS type even when cert-controller is enabled.

Since TLS type Secret needs `tls.crt` and `tls.key` data, we may have to make a  Secret with dummy data (e.g. `dGxzLmNydAo=` ← this is base64 encoded `tls.crt`).